packages/portfolio-contract/src/portfolio.flows.ts

```diff
@@ -488,51 +545,78 @@ export const rebalance = async (
  */
 export const openPortfolio = (async (
   orch: Orchestrator,
-  ctx: PortfolioContext,
+  ctx: PortfolioBootstrapContext,
   seat: ZCFSeat,
   offerArgs: OfferArgsFor['openPortfolio'], // TODO: USDN/USDC ratio
 ) => {
   await null; // see https://github.com/Agoric/agoric-sdk/wiki/No-Nested-Await
   try {
-    const { makePortfolioKit } = ctx;
+    const {
+      makePortfolioKit,
+      zoeTools,
+      axelarChainsMap,
+      chainHubTools,
+      contractAddresses,
+      inertSubscriber,
+    } = ctx;
+    const { destinationEVMChain } = offerArgs;
     const agoric = await orch.getChain('agoric');
     const { chainId } = await agoric.getChainInfo();
     const localAccount: LocalAccount = await agoric.makeAccount();
     const localAccountId: AccountId = `cosmos:${chainId}:${localAccount.getAddress().value}`;
-    const kit = makePortfolioKit(localAccount);
+
+    const nobleChain = await orch.getChain('noble');
+    // Always make a Noble ICA, since we need it for CCTP
+    const nobleAccount = await nobleChain.makeAccount();
+    const nobleAddr = nobleAccount.getAddress();
+
+    // @ts-expect-error LocalAccountMethods vs OrchestrationAccount
+    const kit = makePortfolioKit(nobleAccount, localAccount);
+    kit.keeper.initAave(axelarChainsMap[destinationEVMChain].caip);
+    kit.keeper.initCompound(axelarChainsMap[destinationEVMChain].caip);
```

Comment:
If we were going to init aave and compound every time, we should do that in `makePortfolioKit`; it would simplify the state.

But we're not going to do that every time, right? A trader can open a portfolio with only a USDN position, right? In fact, they can open one with no positions, right?

This also says that their Aave and Compound positions have to be on the same EVM chain. I wasn't expecting that.